src/Kafka/KafkaInstanceDrawer/KafkaDetailsTab.tsx

```diff
@@ -42,15 +66,80 @@ export const KafkaDetailsTab: FunctionComponent<KafkaDetailsTabProps> = ({
       <DetailsTabAlert expiryDate={expiryDate} />
```

Comment:
Based on recent discussions, instead of including the following information in the description list:
> **Type**    Trial

this alert should be updated to say:
> This trial instance will expire in {x} {unit} {y} {unit}